dist/Dockerfile.build/Dockerfile

```diff
@@ -11,11 +11,10 @@ ENV HOME="/root"
 ENV PATH="${HOME}/.cargo/bin:${PATH}"
 
 # build: Rust stable toolchain + 2 previous versions
-RUN curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain 1.45.0 -y && \
-  rustup install 1.42.0 && \
+RUN curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain 1.46.0 -y && \
```

Comment:
That installs 4 Rust versions - 1.43, 1.44, 1.45 and 1.46

Reply:
I'm aware of that. We previously did this to not break CI: as we shift the 3-installed-versions window asynchronous to shifting the 3-tested-versions window. We could stop doing that until we grow back our 3-tested-versions window after [we're cutting it to 1](https://github.com/openshift/release/pull/11523). Shall we?

Reply:
Let stick to one version then and add another one (same as in CPaaS builds) then